### PR TITLE
Support `tinyint(1) unsigned` for `boolean` conversion

### PR DESCRIPTION
**Pull request by lizztheblizz**
Partial fix for #83

### CODE REVIEW

File: cmd/internal/planetscale_edge_database.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/go-sql-driver/mysql"
```
Comment (dbussink) [inline cmd/internal/planetscale_edge_database.go:11]: @lizztheblizz Looks like this was accidentally added?

Comment (lizztheblizz) [reply cmd/internal/planetscale_edge_database.go:11]: Huh. I don't remember touching that at all. Removing.

File: cmd/internal/planetscale_edge_database.go [hunk 2]
```diff
@@ -7,7 +7,6 @@ import (
 	"net/http"
 	"strings"
 	"time"
-
```
Comment (dbussink) [inline cmd/internal/planetscale_edge_database.go:10]: Can you add back the line here to reduce the diff? And in general, these get formatted with an `stdlib` block first etc.

File: cmd/internal/planetscale_edge_database.go [hunk 3]
```diff
@@ -7,7 +7,7 @@ import (
...
+
```
Comment (dbussink) [inline cmd/internal/planetscale_edge_database.go:10]: There's a stray tab now here.

**Review by dbussink — APPROVED** [review]

**Pull request merged**